src/Compilers/Core/Portable/Symbols/IFunctionPointerTypeSymbol.cs

```diff
@@ -5,10 +5,20 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    // PROTOTYPE(func-ptr): Document
+    /// <summary>
+    /// Represents a function pointer type such as "delegate*&lt;void&gt;". Pointer types
+    /// are used only in unsafe code.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
     // https://github.com/dotnet/roslyn/issues/39865: Expose calling convention on either this or IMethodSymbol in general
     public interface IFunctionPointerTypeSymbol : ITypeSymbol
     {
+        /// <summary>
+        /// Gets the signature of the function pointed to by an instance of the function pointer type.
+        /// </summary>
         public IMethodSymbol Signature { get; }
```

Comment:
how is calling convention represented?

Reply:
See the comment above: it is not today.

---
In reply to: [428879107](https://github.com/dotnet/roslyn/pull/44436#discussion_r428879107) [](ancestors = 428879107)